### PR TITLE
Gomod

### PR DESCRIPTION
**Pull request by alexanderturner**
This has been forked on my previous pr due to there being a new external dependency. 

Thanks for your review :)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module github.com/ASE/deb-simple-1
```
Comment (esell) [inline go.mod:1]: should this be `github.com/esell/deb-simple` ?

**Comment by esell** [comment]
Thanks for submitting this! I think we might also need to remove `dep ensure` from the `Makefile` and `travis.yml` files in order to clear out any `dep` related references.

File: go.mod (hunk 1 above)
Comment (alexanderturner) [reply go.mod]: It should

**Comment by esell** [comment]
Now that PR #38 was merged, would you mind re-basing just to get everything lined up again? It looks like a few things got out of whack.

**Pull request merged**